pxr/usd/usdPhysics/parseUtils.cpp

```diff
@@ -0,0 +1,3080 @@
+//
+// Copyright 2024 Pixar
+//
+// Licensed under the terms set forth in the LICENSE.txt file available at
+// https://openusd.org/license.
+//
+
+#include "pxr/pxr.h"
+
+#include "pxr/base/gf/transform.h"
+
+#include "pxr/usd/usdGeom/gprim.h"
+#include "pxr/usd/usdGeom/mesh.h"
+#include "pxr/usd/usdGeom/sphere.h"
+#include "pxr/usd/usdGeom/cube.h"
+#include "pxr/usd/usdGeom/capsule.h"
+#include "pxr/usd/usdGeom/cylinder.h"
+#include "pxr/usd/usdGeom/cone.h"
+#include "pxr/usd/usdGeom/plane.h"
+#include "pxr/usd/usdGeom/points.h"
+
+#include "pxr/usd/usdGeom/metrics.h"
+
+#include "pxr/usd/usdShade/materialBindingAPI.h"
+
+#include "pxr/usd/usdPhysics/parseUtils.h"
+#include "pxr/usd/usdPhysics/parseDesc.h"
+
+#include "pxr/usd/usdPhysics/scene.h"
+#include "pxr/usd/usdPhysics/collisionGroup.h"
+#include "pxr/usd/usdPhysics/materialAPI.h"
+#include "pxr/usd/usdPhysics/rigidBodyAPI.h"
+#include "pxr/usd/usdPhysics/collisionAPI.h"
+#include "pxr/usd/usdPhysics/articulationRootAPI.h"
+
+#include "pxr/usd/usdPhysics/joint.h"
+#include "pxr/usd/usdPhysics/fixedJoint.h"
+#include "pxr/usd/usdPhysics/revoluteJoint.h"
+#include "pxr/usd/usdPhysics/prismaticJoint.h"
+#include "pxr/usd/usdPhysics/sphericalJoint.h"
+#include "pxr/usd/usdPhysics/distanceJoint.h"
+
+#include "pxr/usd/usdPhysics/limitAPI.h"
+#include "pxr/usd/usdPhysics/driveAPI.h"
+#include "pxr/usd/usdPhysics/filteredPairsAPI.h"
+#include "pxr/usd/usdPhysics/meshCollisionAPI.h"
+
+
+#include "pxr/usd/usdGeom/pointInstancer.h"
+
+#include "pxr/base/work/dispatcher.h"
+#include "pxr/base/work/loops.h"
+
+
+PXR_NAMESPACE_OPEN_SCOPE
+
+
+// Gather the filtered pairs from UsdPhysicsFilteredPairsAPI if applied to a prim
+void ParseFilteredPairs(const UsdPrim& usdPrim, SdfPathVector* outFilteredPairs)
```

Comment:
If these are cpp only, should they be in an anonymous namespace and/or prefixed with `_` to align with the coding conventions. 
 https://openusd.org/release/api/_page__coding__guidelines.html#:~:text=OpenUSD%20Code%20Conventions%20and%20Formatting,single%20line%20in%20their%20bodies.